python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -2079,6 +2141,50 @@ async def wait_m1m3_settle(self) -> None:
         self.log.debug("Waiting for m1m3 to settle.")
         await asyncio.sleep(self.m1m3_settle_time)
 
+    async def set_m1m3_slew_controller_settings(
+        self, slew_setting: enum.IntEnum, enable_slew_management: bool
+    ) -> None:
+        """
+        Set a specific M1M3 slew controller setting based on the provided
+        enumeration.
+
+        Parameters
+        ----------
+        slew_setting : enum.IntEnum
+            The specific force component setting to be changed.
+        enable_slew_management : bool
+            True to enable, False to disable the specified force component
+            controlled by the slew controller.
+        """
+        if not isinstance(slew_setting, MTM1M3.SetSlewControllerSettings):
```

Comment:
I think you can remove this check here. Maybe replace the type hint of `slew_setting` to `MTM1M3.SetSlewControllerSettings`?

Reply:
If I remove this check, test_set_m1m3_slew_controller_settings_with_invalid_setting fails.

Reply:
Ok!